src/main/kotlin/no/nav/syfo/lagrevedtak/LagreUtbetaltEventOgPlanlagtMeldingService.kt

```diff
@@ -7,27 +7,34 @@ import java.util.UUID
 import no.nav.syfo.application.db.DatabaseInterface
 import no.nav.syfo.application.metrics.KUN_LAGRET_VEDTAK
 import no.nav.syfo.application.metrics.OPPRETTET_PLANLAGT_MELDING
-import no.nav.syfo.lagrevedtak.db.lagreUtbetaltEvent
+import no.nav.syfo.lagrevedtak.db.lagreUtbetaltEventOgOppdaterStansmelding
 import no.nav.syfo.lagrevedtak.db.lagreUtbetaltEventOgPlanlagtMelding
 import no.nav.syfo.lagrevedtak.db.planlagtMeldingFinnes
 import no.nav.syfo.log
 import no.nav.syfo.model.AKTIVITETSKRAV_8_UKER_TYPE
 import no.nav.syfo.model.BREV_39_UKER_TYPE
 import no.nav.syfo.model.BREV_4_UKER_TYPE
 import no.nav.syfo.model.PlanlagtMeldingDbModel
+import no.nav.syfo.model.STANS_TYPE
 
 class LagreUtbetaltEventOgPlanlagtMeldingService(private val database: DatabaseInterface) {
 
     fun lagreUtbetaltEventOgPlanlagtMelding(utbetaltEvent: UtbetaltEvent) {
+        val planlagtStansmelding = lagPlanlagtMeldingDbModelForUtbetaling(
+            utbetaltEvent,
+            STANS_TYPE,
+            utbetaltEvent.tom.plusDays(17).atStartOfDay().atZone(ZoneId.systemDefault()).withZoneSameInstant(ZoneOffset.UTC).toOffsetDateTime()
```

Comment:
Burde vi spesifisere ZoneId for norge her da vi kanskje ikke vet hva ZoneId.SystemDefautl() vil være?